release_automation.sh

```diff
@@ -1,5 +1,30 @@
 #!/bin/bash
 
+# INSTRUCTIONS FOR TESTING FROM FORKED REPOS
+# Prerequisites: 
+# 1. Fork the following repos to your github user repo:
+#    a) Gutenberg-Mobile: https://github.com/wordpress-mobile/gutenberg-mobile
+#    * .gitmodules on CURRENT_BRANCH should reference your gutenberg fork, replace 'WordPress' with GITHUB_USERNAME 
+#    * (https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/.gitmodules)
+#    b) Gutenberg: https://github.com/WordPress/gutenberg
+#    c) WordPress-Android: https://github.com/wordpress-mobile/WordPress-Android
+#    d) WordPress-iOS: https://github.com/wordpress-mobile/WordPress-iOS
+# 2. Insure that each of your forked repos contains the PR labels specified below:
+GUTENBERG_MOBILE_PR_LABEL="release-process"
+GUTENBERG_PR_LABEL="Mobile App Android/iOS"
+WPANDROID_PR_LABEL="gutenberg-mobile"
+WPIOS_PR_LABEL="Gutenberg integration"
+# 3. Ensure that each of your repos contains the target branch listed below:
+GUTENBERG_MOBILE_TARGET_BRANCH="trunk"
+GUTENBERG_TARGET_BRANCH="master"
+WPANDROID_TARGET_BRANCH="develop"
+WPIOS_TARGET_BRANCH="develop"
+# 4. Update the repo names below to the user repo name for your fork
+GUTENBERG_REPO="WordPress"
+MOBILE_REPO="wordpress-mobile"
+#GUTENBERG_REPO="YOUR_GITHUB_USERNAME"
+#MOBILE_REPO="YOUR_GITHUB_USERNAME"
+
```

Comment:
We can maybe add 5th step to clone the forked `Gutenberg Mobile`repo somewhere near this repo with a different name and enter the relative path to that one when asked